prettyprinter-compat-wl-pprint/prettyprinter-compat-wl-pprint.cabal

```diff
@@ -1,5 +1,5 @@
 name:                prettyprinter-compat-wl-pprint
-version:             1.0.0.1
+version:             1.1
```

Comment:
Why not 1.0.1, or even 1.0.0.2? Widening support should not introduce any breakages.

Reply:
There is breaking change. The compats exported `Data.Monoid ((<>))` not, `Data.Semigroup ((<>))`.